dotnet/src/Skills/Skills.UnitTests/MsGraph/CalendarSkillTests.cs

```diff
@@ -225,16 +225,13 @@ public async Task AddEventAsyncWithoutSubjectFailsAsync()
 
         CalendarSkill target = new(connectorMock.Object);
 
-        // Act
-        var ex = await Assert.ThrowsAsync<ArgumentException>(() => FunctionHelpers.CallViaKernel(target, "AddEvent",
+        // Act & Assert
+        var ex = await Assert.ThrowsAsync<SKException>(() => FunctionHelpers.CallViaKernel(target, "AddEvent",
```

Comment:
nit: What exception is expected?

Reply:
Missing arg throws:
SKException($"Missing value for parameter '{name}'");

(NativeFunction.cs:483)

Reply:
Added an additional check.